doc/source/text.rst

```diff
@@ -306,20 +306,23 @@ The same alignment can be used when ``others`` is a ``DataFrame``:
 Concatenating a Series and many objects into a Series
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-All one-dimensional list-likes can be arbitrarily combined in a list-like container (including iterators, ``dict``-views, etc.):
+An arbitrary combination of ``Series``, ``Index`` and ``np.ndarray`` (1-dimensional) can be passed in a list-like container (including iterators, ``dict``-views, etc.):
 
 .. ipython:: python
 
     s
     u
-    s.str.cat([u.values, ['A', 'B', 'C', 'D'], map(str, u.index)], na_rep='-')
```

Comment:
put some comments here, this is just too much code in a row

Reply:
From  my POV, it's just one line of code, while showing all the ingredients. But I'm gonna change it back to just using variants of `u` in different "boxes".

I think it's fair enough to show just `u` and not `u.values` and `pd.Index(u)` as well.

Reply:
so pls show u.values, I have no idea what it actually is, and as I said, pls either add comments or split up showing s, u, u.values, this is just too long

Reply:
@jreback `u` is shown directly above...?! Also in the screenshot I posted - in context it's IMO very clear (especially as it's at the end of the concatenation section, where the reader has seen the same objects 10 times already).

I can of course show **both** `u` and `u.values`, but I thought that understanding the relationship between a Series and its `.values` would be safe to expect of a users pandas-fu.